### DISCUSSION TITLE
Presentation on quarto at JuliaCon 2022

### DISCUSSION THREAD

**Discussion by dmbates** [quarto-dev/quarto-cli#586] [Feature Requests]
I wrote on the Julia slack that I had been successful in converting to quarto most of a book fitting mixed-effects models with Julia.  It only took a couple of days starting from scratch (as in, read how to install quarto).

I think quarto is going to be very important for the Julia community and it happens that proposals for content at JuliaCon 2022 are open until April 11.  See https://pretalx.com/juliacon-2022/cfp

I would very much like to see a presentation on quarto at JuliaCon (the conference is virtual and all presentations are pre-recorded) and would be happy to facilitate in whatever way I can.

**Comment by jjallaire** [comment]
That's great to hear! It would be terrific to have a chance to present at JuliaCon -- I will plan on submitting a talk. The only hitch is that rstudio::conf is the 27th and 28th so I'd need to present on the 29th (realize that presentations are pre-recorded in any case but I'd want to do Q&A, etc.). I'll note that constraint in my submission and hopefully it will work out.

**Comment by mcanouil** [comment]
Closing as completed, see <https://jjallaire.github.io/quarto-juliacon-2022/#/title-slide> and <https://live.juliacon.org/talk/KKBXAZ>.
